spec/dummy/.controlplane/templates/secrets.yml

```diff
@@ -1,10 +0,0 @@
-kind: secret
```

Comment:
I thought it would be best to remove this file and let `setup-app` handle the automatic creation, since we have to check if the secret/policy exists anyway.

An alternative method would be to keep this file and just call `apply-template secrets -a $APP_NAME` from `setup-app`, but that would leave this file to be a bit redundant.